src/braket/device_schema/quera/quera_ahs_paradigm_properties_v1.py

```diff
@@ -0,0 +1,328 @@
+            local detuning patterns
+        commonDetuningResolution(Decimal): Resolution with which time-dependent
+            term of local detuning can be specified
+        localDetuningResolution(Decimal): Resolution with which site-dependent
+            coefficients can be specified
+        detuningSlewRateMax(Decimal): Maximum slew rate for changing the local detuning
+        numberLocalDetuningSites (int): Max number of sites available for the
+            local Rydberg detuning pattern
+        spacingRadialMin (Decimal): Minimum radial spacing between any two
+            sites in the local detuning pattern
+        timeResolution(Decimal): Resolution with which times for local Rydberg
+            drive parameters can be specified
+        timeDeltaMin(Decimal): Minimum time step with which times for local
+            Rydberg drive parameters can be specified
+    """
+
+    detuningRange: List[Decimal]
+    commonDetuningResolution: Decimal
+    localDetuningResolution: Decimal
+    detuningSlewRateMax: Decimal
+    numberLocalDetuningSites: int
+    spacingRadialMin: Decimal
+    timeResolution: Decimal
+    timeDeltaMin: Decimal
+
+
+class Rydberg(BaseModel):
+    """
+    Rydberg Model
+    Attributes:
+        c6Coefficient (Decimal): Rydberg-Rydberg C6 interaction coefficient
+        timeMax (Decimal): Maximum duration of Rydberg drive
+        rydbergGlobal: Rydberg Global
+        rydbergLocal: Rydberg Local
+    """
+
+    c6Coefficient: Decimal
+    timeMax: Decimal
+    rydbergGlobal: RydbergGlobal
+    rydbergLocal: RydbergLocal
+
+
+class PerformanceRydbergGeometry(BaseModel):
+    """
+    Performance of Rydberg Geometry
+    Attributes:
+        positionErrorAbs (Decimal): Error between target and actual site position
+    """
+
+    positionErrorAbs: Decimal
+
+
+class PerformanceRydbergGlobal(BaseModel):
+    """
+    Performance of Rydberg Global
+    Attributes:
+        rabiFrequencyErrorRel (Decimal): random error in the Rabi frequency, relative
+        rabiFrequencyHomogeneity (Decimal): fractional RMS Rabi frequency
+            non-uniformity across the user area
+        detuningErrorAbs (Decimal): random error in the detuning, absolute
+        phaseErrorAbs (Decimal): random error in the phase, absolute
+        omegaTau (Decimal): Product of Rabi frequency and Rabi decay time
+            for a single qubit under resonant Rydberg drive - average over
+            field of view (no local detuning applied)
+        singleQubitFidelity (Decimal): Single qubit fidelity of a pi(3.14)  pulse
+        twoQubitFidelity (Decimal): Fidelity of two-qubit entangled state
+    """
+
+    rabiFrequencyErrorRel: Decimal
+    rabiFrequencyHomogeneity: Decimal
+    detuningErrorAbs: Decimal
+    phaseErrorAbs: Decimal
+    omegaTau: Decimal
+    singleQubitFidelity: Decimal
+    twoQubitFidelity: Decimal
+
+
+class PerformanceRydbergLocal(BaseModel):
+    """
+    Performance of Rydberg Local
```

Comment:
A better description would be: "Parameters determining the limitations of the shifting field"